Internal Opensearch Support (#3175)
chef/chef-server: docs-chef-io/content/server/upgrades.md

The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.

> #### Upgrading to 14.13

Reviewer: 15.0

> #### Upgrading to 14.13

Reviewer: ```suggestion
#### Upgrading to 15.0
```

> Chef Infra Server 14.13 supports External Opensearch for indexing. Migrating from Elasticsearch to External Opensearch needs ontime reindexing and reconfigure after upgrade. This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.

Reviewer: ```suggestion
Chef Infra Server 15.0 supports external OpenSearch for indexing. Migrating from Elasticsearch to External OpenSearch needs a one-time reindexing and reconfigure after the upgrade. This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.
```

> Chef Infra Server 14.13 supports External Opensearch for indexing. Migrating from Elasticsearch to External Opensearch needs ontime reindexing and reconfigure after upgrade. This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.

Author: Still need to update the docs PR with new version 15.X

{{< note >}}

> Set the `opensearch['external']` & `opensearch['external_url']` attribute in [chef-server.rb]({{< relref "config_rb_server_optional_settings" >}}) to enable external Opensearch for indexing.

Reviewer: ```suggestion
Set the `opensearch['external']` & `opensearch['external_url']` attribute in [chef-server.rb]({{< relref "config_rb_server_optional_settings" >}}) to enable external OpenSearch for indexing.
```
